flight/Modules/Stabilization/stabilization.c

```diff
@@ -84,16 +84,20 @@
 #define HORIZON_MODE_MAX_BLEND               0.85f
 
 enum {
-	PID_RATE_ROLL,   // Rate controller settings
+	PID_RATE,   // Rate controller settings
```

Comment:
@kubark42 That's a good point... how about `PID_GROUP_RATE`? Otherwise I'll probably go with `PID_RATE_GROUP`